packages/runner-ct/cypress/component/SpecList.spec.tsx

```diff
@@ -139,7 +145,7 @@ describe('SpecList', () => {
 
     mount(<Subject />, { styles })
 
-    cy.get('[placeholder="Find spec..."').click()
+    cy.get('[placeholder="Find spec..."]').click()
```

Comment:
Is this only change with significance?

Reply:
No, actually this doesn't fix any known issue. The change at the bottom (from `.contains` to `.should('contain', input)`) is the real fix, as for some reason the retry logic is different between those commands.